docs/rst/amldashboard/user_manual/usage.rst

```diff
@@ -0,0 +1,237 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_usage:
+
+#####
+Usage
+#####
+
+Prerequisites
+=============
+
+Ensure you have installed the *AML Dashboard* using one of the following methods:
+
+- :ref:`Linux Installation <amldashboard_linux>`
+- :ref:`Docker Image Installation <amldashboard_docker>`
+
+Execution
+=========
+
+To run the *AML Dashboard*, follow the steps below:
+
+Backend Server
+--------------
+
+Run the following commands to start the backend server:
+
+1. Navigate to the ``backend`` directory.
+
+.. code-block:: bash
+
+   cd backend
+
+2. Start the server:
+
+.. code-block:: bash
+
+   python3 server.py
+
+AML-IP Nodes
+------------
+
+Depending on your requirements, you can run different types of |amlip| nodes:
+
+1. Load the |amlip| environment
+
+.. code-block:: bash
+
+   source /AML-IP/install/setup.bash
+
+2. Navigate to the ``backend`` directory.
+
+.. code-block:: bash
+
+   cd backend
+
+* **Computing Node**
+
+To run a Computing Node, execute:
+
+.. code-block:: bash
+
+   python3 computing.py
+
+You can run multiple computing nodes simultaneously.
+Each computing node will continuously await job assignments.
+When running multiple nodes, they will collectively distribute the workload, optimizing the time to find a solution.
+
+To stop a computing node, simply run ``ctrl+C``.
+
+* **Model Manager Sender Node**
+
+To run a Model Manager Sender Node, execute:
+
+.. code-block:: bash
+
+   python3 sender.py
+
+To stop a Model Manager Sender Node, simply run ``ctrl+C``.
+
+* **Inference Node**
+
+To run an Inference Node, execute:
+
+.. code-block:: bash
+
+   python3 inference.py
+
+To stop an Inference Node, simply run ``ctrl+C``.
+
+AML Dashboard
+-------------
+
+To run the dashboard, follow the steps below:
+
+1. Navigate to the ``frontend/aml_dashboard`` directory.
+
+.. code-block:: bash
+
+   cd frontend/aml_dashboard
+
+2. Start the dashboard:
+
+.. code-block:: bash
+
+   npm run dev
+
+With the *AML Dashboard* up and running, you can access it at the following address:
+
+.. code-block:: bash
+
+   http://localhost:5173/
+
+Data Management Tab
+^^^^^^^^^^^^^^^^^^^
+
+To create a dataset, simply follow these steps:
+
+1. Activate the ``video`` switch in the webcam section to enable webcam capture.
+
+2. Specify the label of the dataset in the *Instance label* section.
+
+3. Click on the ``Hold to record instances`` button in the *Capture instances to the training set* to start recording instances.
+
+4. Once recorded, the dataset will be promptly displayed in the *dataset browser* section for easy access and management. In order to be able to train a model, you must have at least two classes in the dataset.
```

Comment:
I will add that for the model to function, there are needed at least two classes after step 4 in a note.

Reply:
Done